### PR TITLE
Improve performance of kubeless runtime

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- Improve the performance of python function
- Investigation is available [here](https://github.com/kyma-project/kyma/issues/10154#issuecomment-811858828)

**Related issue(s)**

#10154

### CODE REVIEW

File: components/function-runtimes/python38/kubeless/kubeless.py [hunk 1]
```diff
@@ -85,13 +89,19 @@ def metrics():
     return prom.generate_latest(prom.REGISTRY)
 
 
+@app.error(500)
+def exception_handler():
+    return 'Internet server error'
```
Comment (aerfio) [inline components/function-runtimes/python38/kubeless/kubeless.py:94]: did you mean "internal"?

File: components/function-runtimes/python38/kubeless/kubeless.py [hunk 2]
```diff
@@ -102,26 +112,12 @@ def handler():
...
-                p.terminate()
-                p.join()
-                return bottle.HTTPError(408, "Timeout while processing the function")
-            else:
-                p.join()
-                if isinstance(res, Exception) and not isinstance(res, bottle.HTTPResponse):
-                    logging.error("Function returned an exception: %s", res)
-                    raise res
-                return res
+            return func(event, function_context)
```
Comment (aerfio) [inline components/function-runtimes/python38/kubeless/kubeless.py:120]: The way you changed this means we do not have timeout errors anymore, am I correct?

File: components/function-runtimes/python38/kubeless/kubeless.py (hunk 1 above)
Comment (dbadura) [reply components/function-runtimes/python38/kubeless/kubeless.py:94]: Yea, It was old joke in compass. I will change it.

File: components/function-runtimes/python38/kubeless/kubeless.py (hunk 2 above)
Comment (dbadura) [reply components/function-runtimes/python38/kubeless/kubeless.py:120]: I can add the timeout mechanism.

Comment (dbadura) [reply components/function-runtimes/python38/kubeless/kubeless.py]: @rJankowski93 and I managed the timeout issue

**Review by rJankowski93 — APPROVED** [review]

**Review by pPrecel — APPROVED** [review]

**Pull request merged**